MainApp/MainApp/Resources/Main.xcassets/Register/ic_google.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
```

Comment:
images specific for a certain feature must be put inside its own xcassets inside Feature/<name>/Resources/<name>.xcasset

Reply:
has created new file register.xcasset